README.md

```diff
@@ -1,85 +1,126 @@
 # rciam-federation-registry-agent
 
-***Rciam federation registry agent*** main objective is to sync data between rciam federation registry with satosa and mitreId respectively.
-This python library includes a module named `ServiceRegistryAms/` to pull and publish messages from Argo messaging service using the argo-ams-library,
-an API module named `MitreidConnect/` to communicate with the api of the mitreId.
+**RCIAM Federation Registry Agent** main objective is to sync data between RCIAM Federation Registry and
+Keycloak/MITREid Connect/SimpleSAMLphp.
+This python library includes a module named `ServiceRegistryAms/` to pull and publish messages from ARGO Messaging
+Service using the argo-ams-library, an API module named `MitreidConnect/` to communicate with the API of the MITREid, an
+API module named `Keycloak/` to communicate with the API of the Keycloak.
 The main standalone scripts that are used to deploy updates to the third party services are under `bin/`:
 
-* `deployer_ssp` for simple saml php
-* `deployer_mitreid` for mitreid
+- `deployer_keycloak` for Keycloak
+- `deployer_mitreid` for MITREid
+- `deployer_ssp` for SimpleSAMLphp
 
 ## Installation
 
 First install the packages from the requirements.txt file
+
 ```bash
 pip install -r requirements.txt
 ```
 
-Install rciam-federation-registy-agent
+Install rciam-federation-registry-agent
+
 ```bash
 pip install rciam-federation-registry-agent
 ```
 
 ## Usage
 
-### deployer_ssp
-deployer_ssp requires the path of the ssp config file as an argument
+### deployer_keycloak
+
+deployer_keycloak requires the path of the config file as an argument
+
 ```bash
-$ deployer_ssp -c example_deployers.config.json
+deployer_keycloak -c example_deployers.config.json
 ```
 
 ### deployer_mitreid
-deployer_mitreid requires the path of the ssp config file as an argument
+
+deployer_mitreid requires the path of the config file as an argument
+
 ```bash
-$ deployer_mitreid -c example_mitre.config.json
+deployer_mitreid -c example_deployers.config.json
+```
+
+### deployer_ssp
+
+deployer_ssp requires the path of the config file as an argument
+
+```bash
+deployer_ssp -c example_deployers.config.json
 ```
 
 ## Configuration
-Here is a description with the values that the mitreid and ssp conf file must include
+
+Here is a description with the values that the MITREid and SSP conf file must include
 example_deployers.config.json
```

Comment:
An example of the required configuration file can be found in `conf/example_deployers.config.json`. The different configuration options are described below.